pages/api/student_email_join.js

```diff
@@ -2,14 +2,36 @@ import prisma from '../../prisma/prisma';
 
 export default async function handle(req, res) {
   const data = JSON.parse(req.body);
-  const createdStudentEmail = await prisma.classroom.update({
-    where: {
-      classroomId: data['classId'][0]
-    },
-    data: {
-      fccUserIds: { push: data['email'] }
+  const userInfoReq = await fetch(
+    `http://localhost:3001/get-student-profile?email=${data['email']}`
+  );
+  const userInfo = await userInfoReq.json();
+  if (userInfo.length != 0) {
+    const existingStudent = await prisma.classroom.findMany({
+      where: {
+        classroomId: data['classId'][0]
+      }
+    });
+    const ids = existingStudent[0]['fccUserIds'];
+    if (ids.includes(userInfo[0]['uuid'])) {
+      return res.json(409, {
+        error: 1,
+        msg: 'Account already exists'
+      });
```

Comment:
```suggestion
      return res.status(409).json({
        error: 1,
        msg: 'Account already exists'
      });
```
`res.json()` only accepts a single argument, so everything after 409 is ignored.  If we use `res.status()` first, we can chain the status code into the response

Reply:
Alright sounds good just made the change to all json uses.